orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -66,6 +71,11 @@
     this.metricsConfig = metricsConfig;
 
     TypeDescription orcSchema = ORCSchemaUtil.convert(schema);
+
+    List<Integer> integers = OrcSchemaVisitor.visitSchema(orcSchema, new EstimateOrcAveWidthVisitor());
+    estimateLength = integers
+        .stream().reduce(Integer::sum).orElse(1);
```

Comment:
orElse(0) ?   If there is no field in this orc schema, its estimate length should be 0 , right ?